ote_cli/setup.py

```diff
@@ -1,8 +1,28 @@
+"""
+Setup configuration.
```

Comment:
OTE CLI is more a set of scripts, rather than a package, but what do you think of making it [PEP-561](https://www.python.org/dev/peps/pep-0561/) compliant?

Reply:
don't think that it is needed, at least for now